internal/dev_server/sdk/get_server_flags.go

```diff
@@ -22,7 +22,7 @@ func GetServerFlags(w http.ResponseWriter, r *http.Request) {
 			http.Error(w, "flag not found", http.StatusNotFound)
 		}
 	} else {
-		body = ServerAllPayloadFromFlagsState(allFlags)
+		body = ServerFlagsFromFlagsState(allFlags)
```

Comment:
For PHP, the `/sdk/flags` endpoint should return only the flags in an array, not the full flags + segments payload.